include/benchmark/benchmark.h

```diff
@@ -290,11 +290,50 @@ BENCHMARK(BM_test)->Unit(benchmark::kMillisecond);
 #define BENCHMARK_OVERRIDE
 #endif
 
+#if defined(__GNUC__)
```

Comment:
Not really sure if this outermost guard is needed, but then i don't know what set of per-cpu defines MSVC provides.